common/json.go

```diff
@@ -30,9 +33,20 @@ type JSONHTTPResponse struct {
 	// Headers are the HTTP headers of the response.
 	Headers http.Header
 
-	// Body is the JSON-unmarshalled response body. Aside from the fact
+	// body is the JSON-unmarshalled response body. Aside from the fact
 	// that it's JSON-unmarshalled, it's identical to bodyBytes.
-	Body *ajson.Node
+	// If there were no bytes this will be nil.
+	body *ajson.Node
+}
+
+// GetBody returns JSON node. If it is empty the flag will indicate so.
+// Empty response body is a special case and should be handled explicitly.
+func (j *JSONHTTPResponse) GetBody() (*ajson.Node, bool) {
```

Comment:
Let's just call this Body() (https://go.dev/doc/effective_go#Getters)

Reply:
I like this!